x-pack/plugins/encrypted_saved_objects/server/create_model_version.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mergeTransformFunctions } from '@kbn/core-saved-objects-base-server-internal';
```

Comment:
NIT: so, in theory the `-internal` core packages are, well, internal to core. But it's probably fine, especially given it's supposedly only temporary until the mid/long term proper design.

Reply:
The long term plan is for tighter integration between Core and ESO, and potentially eliminating the need for wrappers. 🤞  But as that roadmap is undefined, would it be worth just moving this function over to `@kbn/core-saved-objects-server` to avoid the import from `-internal`?

Reply:
`@kbn/core-saved-objects-server` is a type-only package. It could maybe be moved to `@kbn/core-saved-objects-utils-server`, but I don't think it's mandatory for this PR